docs/platforms/javascript/guides/aws-lambda/configuration/integrations/contextlines.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: ContextLines
+description: "Adds source code from inline JavaScript of the Node JavaScript files. (default)"
+---
+
+_Import name: `Sentry.contextLinesIntegration`_
+
+This integration adds source code from inline JavaScript of the Node JavaScript files.
```

Comment:
I'm having trouble parsing what's meant by "inline JavaScript of the Node JavaScript files"

Do you mean 
```suggestion
This integration adds source code from inline JavaScript used in Node JavaScript files.
```
or maybe
```suggestion
This integration adds source code from inline JavaScript to the Node JavaScript files.
```

or something else?

Reply:
sorry, this is a bit confusing - I updated it, LMK if it is clearer now...?

Reply:
Yes, much clearer now, thanks!